x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/related_detection_rules_callout.tsx

```diff
@@ -7,32 +7,37 @@
 
 import React, { memo } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
-import { EuiCallOut } from '@elastic/eui';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
 import { useTestIdGenerator } from '../../../../../hooks/use_test_id_generator';
-import { LinkToApp } from '../../../../../../common/components/endpoint/link_to_app';
-import { APP_UI_ID, SecurityPageName } from '../../../../../../../common';
+import { useKibana } from '../../../../../../common/lib/kibana';
 
 export const RelatedDetectionRulesCallout = memo<{ 'data-test-subj'?: string }>(
   ({ 'data-test-subj': dataTestSubj }) => {
     const getTestId = useTestIdGenerator(dataTestSubj);
 
+    const {
+      docLinks: {
+        links: { securitySolution },
+      },
+    } = useKibana().services;
+
     return (
       <EuiCallOut iconType="iInCircle" data-test-subj={getTestId()}>
         <FormattedMessage
-          id="xpack.securitySolution.endpoint.policy.details.detectionRulesMessage"
-          defaultMessage="View {detectionRulesLink}. Prebuilt rules are tagged “Elastic” on the Detection Rules page."
+          id="xpack.securitySolution.endpoint.policy.details.detectionRulesDocsMessage"
+          defaultMessage="The Endpoint Security detection rule is enabled automatically with Elastic Defend. This rule must remain enabled to receive endpoint alerts. {detectionRulesDocsLink}."
```

Comment:
```suggestion
          defaultMessage="The Endpoint Security detection rule is enabled automatically with Elastic Defend. This rule must remain enabled to receive Endpoint alerts. {detectionRulesDocsLink}."
```

Reply:
Made the change here: https://github.com/elastic/kibana/pull/172471/commits/3102e34a843918d0789908e310aa672ec132e7f3